modules/nf-core/kraken2/add/main.nf

```diff
@@ -0,0 +1,38 @@
+process KRAKEN2_ADD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-5799ab18b5fc681e75923b2450abaa969907ec98:87fc08d11968d081f3e8a37131c1f1f6715b6542-0' :
```

Comment:
What is in this mulled container exactly, when the conda environment is just kraken2? I wonder if this is also possibly causing the issue with the `chown` error message?

Reply:
Tried a different container: 

```
   container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
        'https://depot.galaxyproject.org/singularity/kraken2:2.1.2--pl5321h9f5acd7_2' :
        'biocontainers/kraken2:2.1.2--pl5321h9f5acd7_2' }"
```
But the error persists...